legal-api/src/legal_api/resources/nr_type_map.py

```diff
@@ -0,0 +1,199 @@
+            # ('UC', Business.LegalTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestActions.CNV.value),
+            # ('CUL', Business.LegalTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestActions.CHG.value),
+            # ('ULCT', Business.LegalTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestActions.MVE.value),
+            # ('RUL', Business.LegalTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestActions.REST.value),
+            # ('RUL', Business.LegalTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestActions.REH.value),
+            # ('RUL', Business.LegalTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestActions.REN.value),
+            # ('UA', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.AS.value),
+            # ('XUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.NEW_AML.value),
+            # ('XUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.NEW.value),
+            # ('XUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.AML.value),
+            # ('XCUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.CHG.value),
+            # ('XRUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.REST.value),
+            # ('XRUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.REH.value),
+            # ('XRUL', Business.LegalTypes.XPRO_UNLIMITED_LIABILITY_COMPANY.value, RequestActions.REN.value),
+            ('FI', Business.LegalTypes.FINANCIAL.value, RequestActions.NEW.value),
+            ('CFI', Business.LegalTypes.FINANCIAL.value, RequestActions.CHG.value),
+            ('RFI', Business.LegalTypes.FINANCIAL.value, RequestActions.REST.value),
+            ('RFI', Business.LegalTypes.FINANCIAL.value, RequestActions.REH.value),
+            ('RFI', Business.LegalTypes.FINANCIAL.value, RequestActions.REN.value),
+            ('PA', Business.LegalTypes.PRIVATE_ACT.value, RequestActions.NEW.value),
+            ('PAR', Business.LegalTypes.PARISHES.value, RequestActions.NEW.value),
+            ('BC', Business.LegalTypes.BCOMP.value, RequestActions.NEW.value),
+            ('BEAM', Business.LegalTypes.BCOMP.value, RequestActions.AML.value),
+            ('BEC', Business.LegalTypes.BCOMP.value, RequestActions.CHG.value),
+            ('BECT', Business.LegalTypes.BCOMP.value, RequestActions.MVE.value),
+            ('BERE', Business.LegalTypes.BCOMP.value, RequestActions.REST.value),
+            ('BERE', Business.LegalTypes.BCOMP.value, RequestActions.REH.value),
+            ('BERE', Business.LegalTypes.BCOMP.value, RequestActions.REN.value),
+            ('BECV', Business.LegalTypes.BCOMP.value, RequestActions.CNV.value),
+            ('BECR', Business.LegalTypes.COMP.value, RequestActions.CNV.value)
+        ]
+
+        rv = []
+
+        # get list of matching items
+        for item in sorted(request_type_mapping):
```

Comment:
not worth sorting the static info, but making it a runtime, all the time call?

Reply:
I didn't want to sort the static data (so that its order matched where we got it from, to make comparisons with the original code easier).

I removed `sorted()` from here.